docker-compose.yml

```diff
@@ -137,6 +137,30 @@ services:
       - "logging=true"
     restart: always
     logging: *default-logging
+  login-dashboard:
+    image: lblod/acmidm-login-service:0.11.0
+    environment:
+      MU_APPLICATION_AUTH_DISCOVERY_URL: "https://authenticatie-ti.vlaanderen.be/op"
+      MU_APPLICATION_AUTH_CLIENT_ID: "a2c0d6ea-01b4-4f68-920b-10834a943c27"
```

Comment:
I would use the correct ids already, if we have it. Or snip it out so we just configure it on the server. If anything, we should probably add the prod values, but we don't do it for the loket version either.

Reply:
I just copied whatever was set on the login service for loket.

I think the reasoning here is that it is okay to share these values from the QA setup, but not the ones from the production setup. These QA values now work as a nice example of how to setup the overrides for production.

Reply:
> but not the ones from the production setup.

I don't think that's true, since everything is public besides the client secret. I guess it's just done this way because we usually don't have the prod values yet when we merge / deploy.

I don't mind keeping QA values, but it feels strange to keep the _wrong_ ones in there. We could do something like in LPDC if we don't know the values?

https://github.com/lblod/app-lpdc-digitaal-loket/blob/5e85eb2ec37ce1879b5b35f312e314b021e12a17/docker-compose.yml#L145-L150

Reply:
Okay, I'll use the production values for all three. That would make the deploy a little easier. Except for the secret though, but I'll make it a little easier to see that it needs to be replaced.